test/plugin/test_parser_syslog.rb

```diff
@@ -276,7 +276,7 @@ class TestAutoRegexp < self
     def test_auto_with_legacy_syslog_message
       @parser.configure(
                         'time_format' => '%b %d %M:%S:%H',
-                        'mseeage_format' => 'auto',
+                        'meseeage_format' => 'auto',
```

Comment:
`message_format` ?

Reply:
Thank you very much, @repeatedly 
The patch has just been updated as your suggestion.
Have a great day ahead :smile: